data.json

```
@@ -760,5 +760,11 @@
     "githubUsername": "anotherwebguy",
     "imageUrl": "https://user-images.githubusercontent.com/66346161/114270825-18c90d80-9a2c-11eb-9173-1c072e90ac27.png",
     "issueNumber": 436
+  },
+   {
+    "name": "Rishabh Rathore",
+    "githubUsername": "Rishabhrathore055",
+    "imageUrl": "https://user-images.githubusercontent.com/77366500/115426452-234e8880-a21e-11eb-9e2d-c508acdb921f.jp",
```

take a look at your image link there is a missing `p` for `jpg`.

```suggestion
    "imageUrl": "https://user-images.githubusercontent.com/77366500/115426452-234e8880-a21e-11eb-9e2d-c508acdb921f.jpg",
```